cfg/18.09/definitions.yaml

```diff
@@ -0,0 +1,1824 @@
+    scored: false
+  - id: 1.1.2
+    description: "Ensure that the version of Docker is up to date (Not Scored)"  
+    type: skip 
+    audit: "docker version"
+    remediation: |
+      You should monitor versions of Docker releases and make sure
+      your software is updated as required.
+    scored: false
+    
+- id: 1.2
+  description: "Linux Hosts Specific Configuration"
+  checks:
+  - id: 1.2.1
+    description: "Ensure a separate partition for containers has been created (Scored)"
+    audit: "grep /var/lib/docker /proc/mounts"
+    tests:
+      test_items:
+      - flag: "/var/lib/docker"
+        set: true
+    remediation: |
+      For new installations, you should create a separate partition for the /var/lib/docker mount point.
+      For systems which have already been installed, 
+      you should use the Logical Volume Manager (LVM)
+      within Linux to create a new partition.
+    scored: true
+  - id: 1.2.2
+    description: "Ensure only trusted users are allowed to control Docker daemon (Scored)"
+    audit: "getent group docker"
+    type: manual
+    remediation: |
+      You should remove any untrusted users from the docker group. 
+      Additionally, you should not create a mapping of sensitive directories from the host to container volumes."
+    scored: true
+
+  - id: 1.2.3
+    description: "Ensure auditing is configured for the docker daemon (Scored)"
+    audit: "auditctl -l | grep /usr/bin/dockerd"
+    tests:
+      test_items:
+      - flag: "/usr/bin/dockerd"
+        set: true
+    remediation: |
+      You should add rules for the Docker daemon. For example: 
+      Add the line below to the 
+      /etc/audit/audit.rules file: 
+      -w /usr/bin/dockerd -k docker
+      Then, restart the audit daemon using the following command service auditd restart"
+    scored: true
+
+  - id: 1.2.4
+    description: "Ensure auditing is configured for Docker files and directories - 
+    /var/lib/docker (Scored)"
+    audit: "auditctl -l | grep /var/lib/docker"
+    tests:
+      test_items:
+      - flag: "/var/lib/docker"
+        set: true
+    remediation: |
+      You should add a rule for the /var/lib/docker directory. 
+      For example, Add the line as below to the /etc/audit/audit.rules file:
+      -w /var/lib/docker -k docker
+      Then restart the audit daemon. For example, 
+      service auditd restart"
+    scored: true
+
+  - id: 1.2.5
+    description: "Ensure auditing is configured for Docker files and directories -
+    /etc/docker (Scored)"
+    audit: "auditctl -l | grep /etc/docker"
+    tests:
+      test_items:
+      - flag: "/etc/docker"
+        set: true
+    remediation: |
+      You should add a rule for the /etc/docker directory.
+      For example: Add the line below to the /etc/audit/audit.rules file:
+      -w /etc/docker -k docker
+      Then restart the audit daemon. For example: 
+      service auditd restart"
+    scored: true
+
+    # Revisit this check
```

Comment:
Did you need to do something here?

Reply:
Oh, those are from the old version,  I haven't notice that.
I believe it because the file doesn't have to exist.
But, maybe we could change it the be manual, but I think it better be as it is